app/pages/sections/IncidentSection.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.sections
+
+import play.api.libs.json.JsPath
+import pages.QuestionPage
+
+case object IncidentSection extends QuestionPage[Nothing] {
```

Comment:
We've just the approach on sections yesterday, instead of QuestionPage we've changed the Section files to be Section[JsObject]. Might be worth adding that in